.envrc

```diff
@@ -1 +0,0 @@
-use_flake
```

Comment:
I prefer my local override of `.envrc`... I don't think I would be the only person.